marimo/_plugins/ui/_impl/utils/dataframe.py

```diff
@@ -74,6 +74,6 @@ def _get_row_headers_for_index(
             or dtype == "category"
         ):
             name = str(index.name) if index.name else ""
-            return [(name, index.tolist())]  # type: ignore[list-item]
+            return [(name, index.tolist())]
```

Comment:
Ah, it appears the `type: ignore` was actually needed

```suggestion
            return [(name, index.tolist())]   # type: ignore[list-item]
```